buildsystem/buildsystem/zstackbuild.py

```
@@ -4,12 +4,12 @@
 '''
 import sys
 import os.path
-import ConfigParser
+import configparser
```

Ruff (0.12.2)

`configparser` imported but unused

Remove unused import: `configparser`

(F401)